contrib/pg_tde/documentation/docs/setup.md

```diff
@@ -19,7 +19,7 @@ Load the `pg_tde` at startup time. The extension requires additional shared memo
     * On Debian and Ubuntu:    
 
        ```sh
-       sudo systemctl restart postgresql-17
+       sudo systemctl restart postgresql@17-main
```

Comment:
```suggestion
       sudo systemctl restart postgresql.service
```

Reply:
Good comments but this is a merge commit so not the place to fix this. Sorry! Let's fix all of these as a separate PR.